src/routes/(app)/docs/[lang]/[filename]/+server.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs';
+import path from 'path';
+
+export async function GET({ params }) {
+	const { lang, filename } = params;
+	const filePath = path.resolve('docs', lang, filename);
+
+	if (!fs.existsSync(filePath)) {
+		return new Response(JSON.stringify({ error: 'File not found' }), { status: 404 });
+	}
```

Comment:
By the way can't we just `throw error(...)` here?

like in https://github.com/be-BOP-io-SA/be-BOP/blob/30da835beacaaba102d8f1fd0f2b4eec0329bbef/src/routes/(app)/digital-file/raw/%5Bid%5D/%2Bserver.ts#L5-L21